domains/nyll.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "NyllRE",
+        "email": "52743522+NyllRE@users.noreply.github.com"

```

Invalid email, you can safely remove this if you dont want to share it
```suggestion
        "username": "NyllRE"
```